### DISCUSSION TITLE
抵制Wegame从我做起，使用批处理一键启动ChampR和英雄联盟！

### DISCUSSION THREAD

**Discussion by leonyff** [cangzhang/ChampR#161] [General]
感谢大佬做的软件，让我们免受WeGame的荼毒！

1. 新建文件后缀名 `.bat`，文件名称随意 我这里写的是 `一键启动.bat` ，路径随便放，我是放在了这里！
    ![image](https://user-images.githubusercontent.com/38803732/158299481-e1a12807-3e4c-4c0d-9c09-9c3a41149db9.png)
2. 拷贝如下内容到文件中，**注意程序目录修改为自己的**
   ```bat
    chcp 65001
    
    :: 启动 ChampR
    start /d "C:\Program Files\ChampR" ChampR.exe
    TIMEOUT /T  2
    :: 启动英雄联盟
    start /d "F:\WeGameApps\英雄联盟\TCLS" Client.exe
   ```
3. 桌面右键-新建-创建快捷方式 ，对象位置指定刚刚创建的批处理文件！
   ![image](https://user-images.githubusercontent.com/38803732/158300480-2e4adb32-6d53-4d27-804a-6a3a8696ad27.png)
4. 创建完成，右键它点击属性修改下它的打开方式和图标，还可以设置快捷键启动呦
   ![image](https://user-images.githubusercontent.com/38803732/158301553-5bb09af0-9246-4389-87f6-7b25ac9ee3f5.png)
5. OK ，双击它就可以一键启动 ChampR 和 LOL啦！
   ![image](https://user-images.githubusercontent.com/38803732/158301854-4e1907a3-2636-4f99-bd13-8c88b898fc0e.png)

**我这边有个问题，批处理文件启动后应该会自动关闭批处理窗口，但是我运行成功后如下的报错，窗口也不会自动关掉，手动关闭cmd窗口 ChampR也会退出，但是修改批处理文件，调换启动顺序先启动 lol 再启动ChampR 就不会有问题，请大佬指教下！**
![image](https://user-images.githubusercontent.com/38803732/158304716-b611c3bd-b75d-40a9-89d2-952aa260e1ca.png)

**Comment by MTKmin** [comment]
我尝试了一下 但是 批处理打开LOL的时候还是会启动Wegame 并提示官方启动器为Wegame 这样启动只是单纯的纯净模式下的Wegame

**Reply by leonyff** [reply]
B站这个up主，有分享教程，跟着教程修改下配置文件，就可以用批处理启动了！不会再弹出 wegame

[https://www.bilibili.com/video/BV1rD4y1F78n](https://www.bilibili.com/video/BV1rD4y1F78n)

**Comment by topgaer** [comment]
如果UAC开的比较高，默认情况ChampR和LOL都会弹窗请求管理员权限，可以在前面加一行
%1 mshta vbscript:CreateObject("Shell.Application").ShellExecute("cmd.exe","/c %~s0 ::","","runas",1)(window.close)&&exit cd /d "%~dp0"
允许一次之后，两个软件都可以完成启动

**Reply by Conliq** [reply]
不起作用，但是可以在快捷方式的属性—高级里设置“用管理员身份运行”，也可以实现该功能
